openreview/conference/webfield.py

```diff
@@ -323,33 +323,68 @@ def set_reduced_load_page(self, conference_id, invitation, options = {}):
             content = content.replace("var HEADER = {};", "var HEADER = " + json.dumps(header) + ";")
             return self.__update_invitation(invitation, content)
 
-    def set_recruit_page(self, conference_id, invitation, options = {}, reduced_load_id=None):
-
-        default_header = {
-            'title': conference_id,
-            'subtitle': conference_id,
-            'location': 'TBD',
-            'date': 'TBD',
-            'website': 'nourl',
-            'instructions': '',
-            'deadline': 'TBD'
-        }
-
-        header = self.__build_options(default_header, options)
-
+    def set_recruit_page(self, conference, invitation, reduced_load_id=None):
+
+        if conference.use_recruitment_template:
+            accept_message = f'### Thank you for accepting this invitation from {conference.get_homepage_options().get("title")}.\n\n- Log in to your OpenReview account. If you do not already have an account, you can sign up [here](https://openreview.net/signup).\n\n- Ensure that the email address [TODO] that received this invitation is linked to your profile page and has been confirmed.\n\n- Complete your pending [tasks](https://openreview.net/tasks) (if any) for {conference.short_name}.'
+
+            decline_message = f"### You have declined the invitation from {conference.get_homepage_options().get('title')}."
+
+            quota_message = "### In case you only declined because you think you cannot handle the maximum load of papers, you can reduce your load slightly. Be aware that this will decrease your overall score for an outstanding reviewer award, since all good reviews will accumulate a positive score. You can request a reduced reviewer load by clicking the option below:"
+
+            properties = ''
+
+            if 'reduced_quota' in invitation.reply['content']:
+                properties = '''
+    quotaMessage: ''' + json.dumps(quota_message) + '''
+'''
+        
+            new_webfield_content = '''// Webfield component
+return {
+    component: 'RecruitmentForm',
+    version: 1,
+    properties: {
+        header: ''' + json.dumps(conference.get_homepage_options(), indent=2) + ''',
+        acceptMessage: ''' + json.dumps(accept_message) + ''',
+        declineMessage: ''' + json.dumps(decline_message) + ''',
+        ''' + properties + '''
```

Comment:
I think it's better to list all the possible properties and then set some to `null` if they are not needed.